lib/compat/wordpress-6.1/persisted-preferences.php

```diff
@@ -41,7 +41,7 @@ function gutenberg_register_persisted_preferences_meta() {
 	);
 }
 
-add_action( 'init', 'gutenberg_register_persisted_preferences_meta' );
+add_action( 'rest_api_switched_to_blog', 'gutenberg_register_persisted_preferences_meta' );
```

Comment:
What is this hook?

Reply:
@spacedmonkey It only exists on WordPress.com. If you're curious, all of the REST API requests on WordPress.com are proxied through public-api.wordpress.com, so there's some `switch_to_blog()` to make sure the right context is loaded.